source/manual/alerts/rabbitmq-no-consumers-listening.html.md

```diff
@@ -4,75 +4,106 @@ title: 'RabbitMQ: No consumers listening to queue'
 parent: "/manual.html"
 layout: manual_layout
 section: Icinga alerts
-last_reviewed_on: 2019-12-05
+last_reviewed_on: 2020-09-04
 review_in: 6 months
 ---
 
-[Read more about how we use RabbitMQ][rabbitmq]
+## Check that there is at least one non-idle consumer for rabbitmq queue <queue>
 
-We run a check that there is at least one non-idle consumer for a named
-RabbitMQ queue. (For example, for the `email_alert_service` queue.) The queue
-name should indicate the app responsible for consuming the queue.
+For information about how we use RabbitMQ, see [here][rabbitmq_doc].
 
-The Icinga check is performed by connecting to RabbitMQ's admin API, so the
-information given is from Rabbit's point of view.
+## Criticals
 
-Since we send heartbeats every minute (from each instance of the Publishing
-API) at least one of the consumers for each queue should be active every
-minute. To be conservative, the check checks that at least one consumer of the
-queue is running and has been active within the last 5 minutes.
+### `No consumers listening to queue`
 
-If the check succeeds, it will return the most recent time at which activity
-happened on the queue.
+This check reports a critical error when no consumers are listening to the
+queue.
 
-If the check fails due to no consumers having been active recently, it will
-report a critical failure, giving the most recent idle time, and the idle
-times of each consumer connected to RabbitMQ:
+Icinga connects to RabbitMQ's admin API to check that at least one consumer is
+running for a given RabbitMQ message queue. See
+[here][no_consumers_listening_config] for the alert config.
 
-```
-CRITICAL: No activity for X seconds: idle times are [X, Y, Z]
-```
+### `No activity for X seconds: idle times are [X, Y, Z]`
 
-If the check fails due to there being no consumers connected to the queue, it
-will report a critical failure:
+This checks whether the RabbitMQ consumers for a queue have been active in the
```

Comment:
This sentence is repetitive compared to the one above, but the nature of the alert is very different. Can we incorporate the note (below) more prominently here, to emphasise these are difference between the checks? Why do we have two checks?

Reply:
Sure, I've made the note into a paragraph just below the alert description. I've also emphasised what each alert actually means in each description which should make it clear they're distinct. Let me know what you think?